microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -21,43 +21,104 @@ This whitepaper focuses on the desktop version of Microsoft Edge. Parts of the d
+1. Click the **Search suggestions and filters** section.  The **Search suggestions and filters** settings page<!-- edge://settings/searchFilters --> opens.
 
-Typed characters and the websites you visit are stored locally on the device per profile.  To delete the data, go to `edge://settings/clearBrowserData`. In the **Clear browsing data** window, select the **Browsing history** checkbox, and select the **Clear now** button.
+1. Use the toggle **Show me search and site suggestions using my typed characters**:
 
-If Bing is your default search provider and you are signed in, you may delete your searches through the [Microsoft Privacy dashboard](https://account.microsoft.com/privacy/).  You can clear your browsing history and delete websites from appearing as suggestions in the address bar. Go to `edge://history`, and select **Clear browsing data**.  You can delete the data Microsoft collects from the address bar and search suggestions features on Windows 10 and later. Open **Start** > **Settings** > **Privacy** > **Diagnostics & feedback**. In **Delete diagnostic data**, select **Delete**.  All other data is deleted after 36 months.
+   ![The toggle in Edge Settings for "Show me search and site suggestions using my typed characters"](./index-images/search-suggestions-typed-chars.png)
 
-<!-- Deleting diagnostic data could use rewrite -->
+   If you turn off that toggle, your typed characters are no longer sent to your default search provider, but your search queries are still sent to your default search provider, to provide search results.<!--per ui: "If you turn this off, you'll only see suggestions from your favorites and history. Typed characters won't be sent to the search engine used in the address bar." -->
+
+
+<!-- ------------------------------ -->
+#### Microsoft Search in Bing
+
+If you're signed into Microsoft Edge using your work or school account (AAD account), your typed-in characters are also sent to Microsoft Search in Bing, which returns the work- or school-related content or cross-device suggested queries to you.  Your administrator can choose to turn on or off the work- or school-related content or query suggestions entirely for your organization, by using the `AddressBarMicrosoftSearchInBingProviderEnabled` Microsoft Search in Bing (MSB) policy.
+
+If your administrator turns off the `AddressBarMicrosoftSearchInBingProviderEnabled` policy:
+* In Microsoft Edge on your present device, you see the query suggestions that are returned from your default search engine.<!-- do you see "work- or school-related content"? -->
```

Comment:
do you see "work- or school-related content"?

the provided original draft: 
If you are signed in Edge with your work **or school** account (AAD account), then your typed-in characters will also be sent to Microsoft Enterprise Search provider, which returns the work **or school** related **content or** cross-device **suggested queries** to you.  Your tenant admin can choose to turn on or off the **work** related content **or query suggestions** entirely for your organization by via MSB policy of “AddressBarMicrosoftSearchInBingProviderEnabled”.  If your tenant admin choses to turn it off, you will lose the functionality of seeing the **query suggestions** from another device **and** all the **work** related **results**, but still have the **query suggestions** from your local browser **and results** returned from your default search engine.

@dexterqu2024

Reply:
answer from offline: "On [_list item_] 1, if the administrator turns off the toggle (MSB toggle), then the users won't see the work- or school- related content/results - not from the Microsoft Enterprise Search Provider."

Reply:
result:
Your administrator can choose to turn on or off the work- or school-related results or query suggestions entirely for your organization, by using the `AddressBarMicrosoftSearchInBingProviderEnabled` Microsoft Search in Bing (MSB) policy.  If your administrator turns off the `AddressBarMicrosoftSearchInBingProviderEnabled` policy:
* In Microsoft Edge on your present device:
   * You don't see the work- or school- related results from Microsoft Search in Bing.
   * You see the query suggestions that are returned from your default search engine.
* In Microsoft Edge on your other devices:
   * You don't see the work- or school-related results.
   * You don't see the query suggestions that you previously searched.

Reply:
it would be nice to make the list parallel, strictly consistent.  @dexterqu2024 @captainbrosset 
i'm pushing this commit